package.json

```diff
@@ -32,7 +32,7 @@
     "@theforeman/builder": "^4.14.0",
     "@theforeman/eslint-plugin-foreman": "^4.14.0",
     "@theforeman/stories": "^4.14.0",
-    "@theforeman/test": "^4.14.0",
+    "@theforeman/test": "^6.0.0",
```

Comment:
@sharvit should I just update all of the `@theforeman` packages to 6.0.0 or just the test package?

Reply:
well, you depend only on the `test` right? so I'd say bump only the required package and if that needs newer versions of the others, right?

Reply:
And, you probably do not need packaging PR if you don't bump the others if I'm not mistaken ...

Reply:
@theforeman/packaging any packaging PR needed for just the bump in test package?

Reply:
We should use the same versions for all @theforeman/* packages for comparability (they require each other).
Anyway, only the `test` package contains actual changes.

Reply:
What probably should happen is that any time you depend on a feature introduced in 6.0.1, you actually update the dependency to 6.0.1. In practice, people often forget this.